packages/neon/neon/lib/src/models/app_ids.dart

```diff
@@ -0,0 +1,6 @@
+abstract class AppIDs {
```

Comment:
Enum?

Or because other apps need to extend this?

Reply:
You need to have it as a string value for comparison and with an enum using a field it would always be `AppIDs.news.value` or something like that. I don't see the benefit there

Reply:
Makes sense.

Can we at least make the class `final` or even `sealed`?
I don't think an app needs to extend it.